src/main/kotlin/step2/calculator/Calculator.kt

```diff
@@ -0,0 +1,79 @@
+package step2.calculator
+
+/**
+ * 문자열 계산기
+ *
+ * 사용자가 입력한 문자열 값에 따라 사칙 연산을 수행할 수 있는 계산기를 구현해야 한다.
+ * 문자열 계산기는 사칙 연산의 계산 우선순위가 아닌 입력 값에 따라 계산 순서가 결정된다. 즉, 수학에서는 곱셈, 나눗셈이 덧셈, 뺄셈 보다 먼저 계산해야 하지만 이를 무시한다.
+ * 예를 들어 "2 + 3 * 4 / 2"와 같은 문자열을 입력할 경우 2 + 3 * 4 / 2 실행 결과인 10을 출력해야 한다.
+ *
+ * @since 2023/10/23
+ * @author KimJungSik
+ * */
```

Comment:
이런 주석은 개인적으로 비권장드립니다. 
일단 since나 author는 형상관리시스템이 들어오면서 git blame만 봐도 알 수 있기에 불필요한 라인 및 시간 소비이기에
이보단 class의 역할만 간단하게 main description에 작성해주시면(Ex: 문자열 계산기) 어떨까 싶네요.